docs/architecture/2026-07-30-footwear-virtual-tryon.html

```
@@ -0,0 +1,108 @@
+<title>Footwear in Virtual Try-On — Retire the Exclusion</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)